index.html

```diff
@@ -164,14 +184,34 @@ <h2>Becoming a full member of a section</h2>
         JoiningRelocateCandidate["JoiningRelocateCandidate<br />(Take RelocatedInfo)"]
         style JoiningRelocateCandidate fill:#f9f,stroke:#333,stroke-width:4px
 
-        InitialSendConnectionInfoRequest["RELOCATED_INFO = RelocatedInfo<br/><br/>For all elders in RelocatedInfo.section_info:<br/>send_rpc(ConnectionInfoRequest)<br/><br/>schedule(TimeoutRefused)"]
+        InitialSendConnectionInfoRequest["RELOCATED_INFO = RelocatedInfo<br/>
+        <br/>
+        For node in RELOCATED_INFO.section_info:<br/>
+        send_rpc(ConnectionInfoRequest) to node<br/>
+        <br/>
+        schedule(TimeoutRefused)<br/>
+        schedule(TimeoutResendInfo)"]
         InitialSendConnectionInfoRequest-->LoopStart
 
         LoopStart --> WaitFor
         WaitFor(("Wait for 0:"))
         LocalEvent((Local<br />Event))
         WaitFor --> LocalEvent
 
+        LocalEvent--"TimeoutResendInfo triggered"--> CheckResend
+        CheckResend["(connected, unconnected) =<br/>get_connected_and_unconnected(<br/>RELOCATED_INFO.section_info)<br/>
+        <br/>
+        For node in unconnected:<br/>
+        send_rpc(ConnectionInfoRequest) to node<br/>
+        <br/>
+        For node in connected:<br/>
+        if !has_resource_proof_for(node):<br/>
+        send_rpc(<br/>Rpc::CandidateInfo from RELOCATED_INFO)<br/>to node<br/>
+        <br/>
+        schedule(TimeoutResendInfo)"]
+        CheckResend --> LoopEnd
+
+
         LocalEvent -- ResourceProofForElderReady --> SendNextResourceProofPartForElder
         SendNextResourceProofPartForElder["send_rpc(<br />Rpc::ResourceProofResponse{<br />get_next_resource_proof_part(<br />elder
         name)})"]
```

Comment:
I'm wondering if no changes to the sequence diagram at the end of the document is required after this?

Reply:
Yes, I forgot to update the sequence diagram